tests/h/routes_test.py

```diff
@@ -79,6 +79,7 @@ def test_includeme():
              traverse='/{id}'),
         call('api.profile', '/api/profile'),
         call('api.debug_token', '/api/debug-token'),
+        call(u'api.group_member', u'/api/groups/{pubid}/members/me', factory=u'h.models.group:GroupFactory', traverse=u'/{pubid}'),
```

Comment:
Ah yes. That was a case of careless copy-pasting. I’ve removed them, and updated the comment at the top as a warning to future me.